samples/weather_function/Dockerfile

```diff
@@ -0,0 +1,38 @@
+FROM openjdk:8u171-jdk-stretch as builder
+WORKDIR /opt
+ENV KOTLIN_NATIVE_VER "0.7.1"
```

Comment:
Maybe 0.8?

Reply:
Looking to move to 0.8 provided it is feasible to migrate from Curl to the new Kotlin Native HTTP Client library.